scripts/wheels/linux/build.sh

```diff
@@ -2,13 +2,30 @@
 # PYTHONS=("cp36" "cp37" "cp38" "cp39" "cp310")
 PYTHONS=("cp38")
 export KRATOS_VERSION="9.0.0"
+export KRATOS_BUILD_TYPE="Custom"
+
+export HASH=$(git show -s --format=%h) # Used in version number
+if [[ "${KRATOS_BUILD_TYPE}" != "Release" ]]; then
+    DATE=$(date '+%Y%m%d')
+    export KRATOS_VERSION="${KRATOS_VERSION}.dev${DATE}"
```

Comment:
including dev in the version name explicitly distinguishes from the actual release and is e.g. not installed using standard pip install command - requires pip install --pre.

The date would still allow to have e.g. a nightly version that can be updated consistently using pip install.